scripts/deploy-cloudflare-workers.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Staging deploys
+if git diff --name-only origin/main | grep '^workers/' > /dev/null; then
+  echo "[deploy-cloudflare-workers] Change detected. Deploying to staging..."
+
+  ENV="staging"
+  DEPLOY=true
+
+# Production Deploys
+elif [[ "$CIRCLE_BRANCH" == "release" ]]; then
+  if git diff --name-only HEAD~1 | grep '^workers/' > /dev/null; then
+    echo "[deploy-cloudflare-workers] Change detected. Deploying to production..."
+
+    ENV="production"
+    DEPLOY=true
+  fi
+else
+  echo "[deploy-cloudflare-workers] No change detected. Skipping deployment."
+fi
+
+# Only deploy if a change was detected
+if [ "$DEPLOY" = true ]; then
+  yarn wrangler deploy --config=workers/test/ciTestWorker.toml --env $ENV
+fi
```

Comment:
I think it would simplify things to call this with an explicit `staging` or `production` argument from `.circleci/config.yml`.